pkg/virt-handler/device-manager/pci_device_test.go

```diff
@@ -107,10 +115,57 @@ pciHostDevices:
 		// It's assumed here that it will find a PCI device at 0000:00:00.0
 		pciDevices := discoverPermittedHostPCIDevices(supportedPCIDeviceMap)
 		devs := constructDPIdevices(pciDevices[fakeName], iommuToPCIMap)
-		Expect(devs[0].ID).To(Equal(fakeIommuGroup))
+		Expect(devs[0].ID).To(Equal(strings.Join([]string{fakeIommuGroup, fakeAddress}, deviceIDSeparator)))
 		Expect(devs[0].Topology.Nodes[0].ID).To(Equal(int64(fakeNumaNode)))
 	})
+
+	It("Should validate multiple devices in IOMMU group", func() {
+		By("making sure the environment has a PCI device at " + secondFakeAddress)
+		_, err := os.Stat("/sys/bus/pci/devices/" + secondFakeAddress)
```

Comment:
Do we have a way to change the sysfs? I know that this was done in this way from the previous tests, but if we don't have this address then the new test isn't run

Reply:
I think it's reasonable to assume that both addresses exist on the host but maybe we can try to list the directory content and use existing  addresses on the host. WDYT? 
Can you trigger the PR tests so we can see if the test was skipped on the CI system?

Reply:
I just triggered them.
Either using existing addresses, or if it will be possible to fake the sysfs directories.

Reply:
Thank you. Looks like all the device-manager tests passed on the CI.

Reply:
@andreyod still, it might exist on that particular host, but on another CI system not having that PCI address and the test is simply skipped. If there are any issues we won't be able to catch them

Reply:
Updated the PCI unit tests to use the PCI addresses that exist on the host.